src/filter/distort0r/distort0r.c

```diff
@@ -66,6 +69,16 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
       info->type = F0R_PARAM_DOUBLE;
       info->explanation = "The frequency of the plasma signal";
       break;
+    case 2:
+      info->name = "Mode";
+      info->type = F0R_PARAM_BOOL;
+      info->explanation = "\'Time Based\' or \'Adjustable Velocity\'";
```

Comment:
It is good to think about the possibility of more modes in the future. In
that case, it would make sense to change param type to string, and use
keywords as described in the explanation. Thank you for the quick update!


On Mon, May 9, 2016 at 1:16 AM d-j-a-y <notifications@github.com> wrote:

> In src/filter/distort0r/distort0r.c
> <https://github.com/dyne/frei0r/pull/1#discussion_r62462621>:
>
> > @@ -66,6 +69,16 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
> >        info->type = F0R_PARAM_DOUBLE;
> >        info->explanation = "The frequency of the plasma signal";
> >        break;
> > +    case 2:
> > +      info->name = "Mode";
> > +      info->type = F0R_PARAM_BOOL;
> > +      info->explanation = "\'Time Based\' or \'Adjustable Velocity\'";
>
> Mode with this explanation does not make it clear...
>
> Firstly, i thought to do a "Use Velocity.", then i make a choice (wrong
> one), thinking of future, when more Mode could be implemented.... but KISS
> is the way.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/dyne/frei0r/pull/1/files/e67d2fc566c661fce055cbccce0e7b1f838752e2#r62462621>
>